src/main/java/fact/extraction/AboveThreshold.java

```diff
@@ -37,25 +42,29 @@ public Data process(Data input) {
 
         final int npix = (Integer) input.get("NPIX");
         final int roi = (Integer) input.get("NROI");
-        final short[] timeSerieses = (short[]) input.get(dataKey);
+        final short[] timeSeries = (short[]) input.get(dataKey);
```

Comment:
I can not wrap my head around this. There is no sense in a single time series for all the independent pixels. Why not having a plural 's'?